satinfo/cris-fsr_npp/readme

```diff
@@ -0,0 +1,4 @@
+# start date 2017102000 (NCEP ops 2018021212)
+# NPP out because of mid-wave failure after 2019032518.  LW/SW data returns 2019041618 but mid-wave channels missing 
+# Switch to side-2 electronics after 2019062412 - leave all passive until 2019062600 to adjust to side-2 (?)
+# off after 2021052118, on again 2021100718 for NASA but not here since it is still off in NOAA ops
```

Comment:
nitpick. if branch GMAO is supposed to approximate GMAO choices. Wouldn't you follow NASA choices instead of ops? 
I am ok with either options.

Reply:
I just was nervous about making choices that don't asymptote with what is used in operations now